recipe/meta.yaml

```diff
@@ -9,9 +9,8 @@ source:
   sha256: 325f75eb80fbc5371136e37f323445309ca9f65b6c6f718d0d0e2189e5de1224
 
 build:
-  number: 1
+  number: 7
   script:
-    - rm -f skimage/run-hessian.py
```

Comment:
Do we want this script?

Reply:
I don't even think this script exists... @jni, comments?

Reply:
haha no, this was a hack to fix my botched 0.13.0 release, where I accidentally included a random script in my working directory. You can find the script if you download 0.13.0 from PyPI. :joy: